tests/test_updatediff.py

```diff
@@ -132,11 +132,11 @@ def test_updatediff(tmp_path_factory):
     # Check it's copied OK
     assert answers.read_text() == dedent(
         f"""\
-            # Changes here will be overwritten by Copier
-            _commit: v0.0.1
-            _src_path: {bundle}
-            author_name: Guybrush
-            project_name: to become a pirate
+        # Changes here will be overwritten by Copier
+        _commit: v0.0.1
+        _src_path: {bundle}
+        author_name: Guybrush
+        project_name: to become a pirate\n
```

Comment:
I think all you really needed was to add the last `\n`, right? If so, please do that (to reduce the diff).

Reply:
Sure, I deemed it was not that much noise to bring consistency on multiline-strings indentation. Feel free to do that in another commit/PR.